lib/rom/sql/schema/column_inferrer.rb

```diff
@@ -0,0 +1,82 @@
+require 'rom/sql/types/pg'
```

Comment:
something like this https://gist.github.com/flash-gordon/901b94a9af623952f1babca50d7b8cd9 /cc @solnic can have some thought on this too